UnitedIncome/components
# AccountCircleIllustration -> TypeScript

### Description
> Please provide a brief description of your changes below.

Updates the `AccountCircleIllustration` to use Typescript. Running into some issues with this, mostly to do with aliased imports in tests though.

I've tried the following configs (this component uses `~constants/js/colors.js`)

* The following config errors the Typescript compiler in tests.
```
    "baseUrl": "./",                  
    "paths": {
      "~components": ["components"],
      "~proptypes": ["proptypes"],
      "~constants": ["constants"]
    },          
```

* The following errors Babel by the looks of things (It also silences intellisense)
```
    "baseUrl": "./",                  
    "paths": {
      "~components": ["components"],
      "~proptypes": ["proptypes"],
      "~constants/js/*": ["constants/js/*"]
    },          
```

The module aliasing occurs in the `.babelrc` configuration previously. 


### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Run `yarn start` and give it a try.

### Comments
> Any additional comments that will aid with the review process.

I'm going to keep trying to fix this later, but if anyone else wants to take a stab please feel free. I have a few other things I need to wrap up today before I can dedicate more time to this.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-*)
